scripts/deployWFAIR.js

```diff
@@ -0,0 +1,76 @@
+/* ./scripts/deployWFAIR.js */
+import { transfers } from './utils/transfers';
+require('log-timestamp');
+const hre = require('hardhat');
+const toBN = hre.web3.utils.toBN;
+const fs = require('fs');
+
+const Q18 = (toBN('10')).pow(toBN('18'));
+
+// Load the deployment configuration file and set up constants
+const network = hre.hardhatArguments.network;
+console.log('Script is running on ' + network);
+const actionsFilepath = './scripts/' + network + '/logs/actions.json';
+const configFilepath = './scripts/' + network + '/deployWFAIR.config.json';
+
+let deployConfig;
+try {
+  deployConfig = JSON.parse(fs.readFileSync(configFilepath, 'utf-8'));
+  console.log('Configuration is retrieved from ' + configFilepath);
+} catch (err) {
+  console.error(err.message);
+}
+
+const actions = {};
+actions.transfers = [];
+console.log('Actions will be logged to ' + actionsFilepath);
+
+// In the config file we use WFAIR, but in the transactions we use wei <-- perhaps wei everywhere is better
+const WALLFAIR_TOTAL_SUPPLY = Q18.mul(toBN(deployConfig.wallfairTotalSupply)).toString();
```

Comment:
why `toString()`? BN should be accepted in deploy method

Reply:
Sure. I've moved it down to the next line where the total is displayed.